warehouse/accounts/services.py

```diff
@@ -75,6 +76,35 @@ def check_password(self, userid, password):
 
         return True
 
+    def create_user(self, username, name, password, email,
+                    is_active=False, is_staff=False, is_superuser=False):
+        user = User(username=username,
+                    name=name,
+                    password=password,
+                    is_active=is_active,
+                    is_staff=is_staff,
+                    is_superuser=is_superuser)
+        user.last_login = datetime.datetime.now()
+        self.db.add(user)
+        self.db.flush()
+        email_object = Email(email=email, user_id=user.id,
+                             primary=True, verified=False)
+        self.db.add(email_object)
+        return user
+
+    def update_user(self, user_id, **changes):
+        user = self.get_user(user_id)
+        for attr, value in changes.items():
+            setattr(user, attr, value)
+        self.db.add(user)
+        return user
+
+    def verify_user(self, user_id):
```

Comment:
I think this would be better if it was ``verify_email(self, user_id, email_address)``, then this should just work for a future where we have multiple email addresses and we need to verify them independently.